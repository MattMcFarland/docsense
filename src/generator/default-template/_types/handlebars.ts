export interface BlockHelper {
  fn: (ctx: any) => any;
}
