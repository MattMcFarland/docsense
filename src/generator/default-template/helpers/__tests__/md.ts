import * as Handlebars from 'handlebars';

describe('hbs helpers > md', () => {
  test('matches snapshot', () => {
    Handlebars.registerHelper('md', require('../md'));
    const template = Handlebars.compile(`<div>{{#md}} {{stuff}} {{/md}}</div>`);
    const result = template({ stuff: `#heading \n\n**yay markdown**` });
    expect(result).toMatchSnapshot();
  });
});
