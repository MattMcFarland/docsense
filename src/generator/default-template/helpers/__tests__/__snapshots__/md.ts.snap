// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hbs helpers > md matches snapshot 1`] = `"<div><p class=\\"lh-copy\\"> #heading </p><p class=\\"lh-copy\\"><strong>yay markdown</strong> </p></div>"`;
